### PR TITLE
Migrate to grape

### PR DESCRIPTION
**Pull request by marcinjak**
Migrate rails API to grape API. Also change active_model_serializers to Grape Entities. 

Remove specs for old rails controllers. Addd specs for new grape api requests

### CODE REVIEW

File: app/controllers/api/v1/entities/prop.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        def body
+          sanitize object.body, tags: %w(img)
+        end
+
+        def is_upvote_possible
+          object.propser_id != current_user.id && !user_has_upvoted?
+        end
+
+        def user_has_upvoted?
+          if object.respond_to?(:user_has_upvoted?)
```
Comment (jniechcial) [inline app/controllers/api/v1/entities/prop.rb]: Why is this?

Comment (marcinjak) [inline app/controllers/api/v1/entities/prop.rb]: It's copy-paste from active_model_serializers which I replaced. I tried not to change the logic :)

**Comment by jniechcial** [comment]
:+1:

**Comment by nekath** [comment]
merged #27

**Pull request merged**